llvm/docs/HowToCrossCompileLLVM.rst

```diff
@@ -1,215 +1,236 @@
 ===================================================================
-How To Cross-Compile Clang/LLVM using Clang/LLVM
+How to cross-compile Clang/LLVM using Clang/LLVM
 ===================================================================
 
 Introduction
-============
+------------
 
 This document contains information about building LLVM and
-Clang on host machine, targeting another platform.
+Clang on a host machine, targeting another platform.
 
 For more information on how to use Clang as a cross-compiler,
 please check https://clang.llvm.org/docs/CrossCompilation.html.
 
-TODO: Add MIPS and other platforms to this document.
+This document describes cross-building a compiler in a single stage, using an
+existing ``clang`` install as the host compiler.
 
-Cross-Compiling from x86_64 to ARM
-==================================
+.. note::
+  These instructions have been tested for targeting 32-bit ARM, AArch64, or
+  64-bit RISC-V from an x86_64 host. But should be equally applicable to any
+  other target.
```

Comment:
maybe this should mention linux specifically here. It IS possible to cross-compile from windows to linux, from mac to linux etc, but I don't think this document covers that.

Reply:
Thanks, I've clarified this is targeted at a Linux host.